extension/linkify-urls-in-code.js

```diff
@@ -28,7 +28,7 @@ window.linkifyURLsInCode = (() => {
 		.toArray()
 		.forEach(blob => {
 			if (hasURL(blob.innerHTML)) {
```

Comment:
I don't understand why we have this check. Instead we could just remove it and let `match()` decide whether or not to run the loop depending on the result count.